plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduWithDisabledInferSchemaConnectorTest.java

```diff
@@ -31,7 +37,16 @@
     @Test
     public void testListingOfTableForDefaultSchema()
     {
-        assertQuery("SHOW TABLES FROM default", "VALUES ('customer'), ('nation'), ('orders'), ('region')");
+        // Test methods may run in parallel and create tables in the default schema
+        // Assert at least the TPCH tables exist but there may be more
+        List<String> rows = new ArrayList<>(computeActual("SHOW TABLES FROM default").getMaterializedRows())
+                .stream()
+                .map(row -> ((String) row.getField(0)))
+                .collect(toUnmodifiableList());
+        assertThat(rows).containsAll(
+                REQUIRED_TPCH_TABLES.stream()
+                        .map(TpchTable::getTableName)
+                        .collect(Collectors.toList()));
```

Comment:
nit: Any specific reason on not using similar `toUnmodifiableList`here ?

Reply:
Doesn't seem like it matters given the list clearly cannot be modified.